.github/CODEOWNERS

```diff
@@ -247,6 +247,7 @@
 /pkg/config/remote/service/meta/        @DataDog/remote-config @DataDog/software-integrity-and-trust
 /pkg/diagnose/                          @Datadog/container-integrations
 /pkg/diagnose/connectivity/             @DataDog/agent-shared-components
+/pkg/dynamic-instrumentation/           @DataDog/debugger
```

Comment:
Generally I think for Go, multi-term package names should be all lower-case with no special characters so I think this should be `/pkg/dynamicinstrumentation/` or a composite if it makes sense (`pkg/instrumentation/dynamic`). With that said, our codebase isn't quite consistent about this but we should strive to make it better moving forward.

Reply:
That works for me, fixed.